src/views/Admin/components/CliftonStrengthsUpload/index.js

```diff
@@ -9,21 +9,20 @@ import {
   TableHead,
   TableRow,
 } from '@material-ui/core';
-import { DateTime } from 'luxon';
+import GordonLoader from 'components/Loader';
+import SpreadsheetUploader from 'components/SpreadsheetUploader';
+import { addDays, parseISO } from 'date-fns';
 import { useState } from 'react';
-import { gordonColors } from 'theme';
-import GordonLoader from '../../../../components/Loader';
-import SpreadsheetUploader from '../../../../components/SpreadsheetUploader';
-import CliftonStrengthsService from '../../../../services/cliftonStrengths';
+import CliftonStrengthsService from 'services/cliftonStrengths';
+import styles from './CliftonStrengthsUpload.module.scss';
 import CliftonStrengthsUploadTemplate from './cliftonStrengthsUploadTemplate.csv';
-
-const headerStyle = {
-  backgroundColor: gordonColors.primary.blue,
-  color: gordonColors.neutral.grayShades[100],
-};
+import UploadFailedSVG from './uploadFailed.svg';
+import UploadSuccededSVG from './uploadSucceded.svg';
 
 const successResults = ['Success', 'Added', 'Modified'];
 
+const initDate = parseISO('1900-01-01T00:00:00');
```

Comment:
What was the reasoning for using `date-fns/parseISO` instead of `new Date()` here?

Reply:
Nothing particular, just a decision.